### ISSUE TITLE
Doesn't handle slashes in branch names?

### ISSUE THREAD

**Issue by lamontadams** [iamtheyammer/branch-env-vars#5]
EDIT - OK so I was doing something stupid. ;)

It does look like this doesn't handle slashes in branch names... if I run this action from a branch named `la/ci-testing` I'd expect EXAMPLE to be set to "valueforstaging" but instead it seems to fall through to `!default`

```
      - name: Set branch-based environment variables
        uses: iamtheyammer/branch-env-vars@v1.0.2
        with:
          EXAMPLE: |
            master:valueformaster
            la/ci-testing:valueforstaging
            !default:defaultvalue

      - name: Output
        run: |
          echo "${{ env.EXAMPLE }}"
```

It does seem to still emit this warning when it runs ` Warning: Unexpected input(s) 'EXAMPLE', valid inputs are ['bevOverwrite', 'bevActionOnNoRef', 'bevSetEmptyVars']`

If that's expected then maybe it should be added to the readme?

Thanks.

**Comment by iamtheyammer** [comment]
The warning is expected and is listed at the bottom of the readme. It comes from actions only expecting certain keys that you define in your yml file, but we need more: one for each var, and I can't predict what your var names will be. Don't worry, it still works as expected. 

I'll look into the slashes issue and get back to you today.

**Comment by lamontadams** [comment]
OK I see it in the readme now, I was misreading it before. Not my finest hour today.... ;)

**Comment by iamtheyammer** [comment]
Works now :) Switched from using `.split("/")` to`.indexOf(...)` which keeps all characters past the first slash. Let me know if you have issues.
![image](https://user-images.githubusercontent.com/32084206/123522323-92b37f00-d68a-11eb-9249-085a39a43596.png)

**Comment by lamontadams** [comment]
Thanks, I'll guinea pig it for you sometime this week.

**Comment by Zurielevi** [comment]
Hi,

The latest tag (1.0.3) didn't support this fix (because 1.0.3 version is from 20.5.2021, and your fix was at 26.6.2021)

Can you please make a new tag?

Thanks

**Comment by iamtheyammer** [comment]
Created a new tag (`v1.0.4`). Sorry for the delay!
